README.md

```diff
@@ -559,6 +560,26 @@ program.add(Source::Imm, Arch::X64).set_dst(1).set_imm(0x605).push()
 Again, please refer to [the source and related tests](src/insn_builder.rs) to
 get more information and examples on how to use it.
 
+## Build features
+
+### `no_std`
+
+The `rbpf` crate has a Cargo feature named "std" that is enabled by default. To
+use `rbpf` in `no_std` environments this feature needs to be disabled. To do
+this, you need to modify your dependency on `rbpf` in Cargo.toml to disable the
+enabled-by-default features.
+```toml
+[dependencies]
+rbpf = { version = "1.0", default-features = false }
+```
+Note that when using this crate in `no_std` environments, the `assembler`,
```

Comment:
Nit: Blank lines around the literal block, please

````suggestion
enabled-by-default features.

```toml
[dependencies]
rbpf = { version = "1.0", default-features = false }
```

Note that when using this crate in `no_std` environments, the `assembler`,
````